pg_sparse/sql/_bootstrap_quickstart.sql

```diff
@@ -0,0 +1,76 @@
+DROP TABLE IF EXISTS mock_items;
```

Comment:
@neilyio is this a file which can go in the shared crate as well?

Reply:
No, this is extension-specific SQL